tests/test_get_replica_connections.py

```
@@ -0,0 +1,90 @@
+                self.connection = MagicMock()
+                self.sentinel = None
+                self.service_name = "mymaster"
+
+            def get_replica_connections(self):
+                if hasattr(self, 'sentinel') and self.sentinel is not None:
+                    try:
+                        replica_hostnames = self.sentinel.discover_slaves(service_name=self.service_name)
+                        if len(replica_hostnames) == 0:
+                            raise ConnectionError("Unable to get replica hostname.")
+                        return [(host, int(port)) for host, port in replica_hostnames]
+                    except redis.RedisError as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)